Tools/SandboxTest.ps1

```diff
@@ -75,7 +79,7 @@ $script:UiLibsHash_v2_8 = '249D2AFB41CC009494841372BD6DD2DF46F87386D535DDF8D9F32
 $script:UiLibsHash_NuGet = '6B62BD3C277F55518C3738121B77585AC5E171C154936EC58D87268BBAE91736'
 
 # File Paths
-$script:AppInstallerDataFolder = Join-Path -Path $env:LOCALAPPDATA -ChildPath 'Packages' -AdditionalChildPath $script:AppInstallerPFN
+$script:AppInstallerDataFolder = Join-Path -Path (Join-Path -Path $env:LOCALAPPDATA -ChildPath 'Packages') -ChildPath $script:AppInstallerPFN
```

Comment:
Does PS5 support arrays as the child path?
```powershell
Join-Path -Path $env:LOCALAPPDATA -ChildPath @('Packages'; $script:AppInstallerPFN)```

Reply:
No; `-ChildPath` requires a string. Though one can do something like:

```pwsh
[System.IO.Path]::Combine($env:LOCALAPPDATA, 'Packages', $script:AppInstallerPFN)
```

I just went with `Join-Path` since it was being used everywhere else

Reply:
It does seem that this support was added recently though (for PS7 of course) :D

- https://github.com/PowerShell/PowerShell/pull/24677

According to the Docs PR, that's available from PowerShell 7.6-preview.4

- https://github.com/MicrosoftDocs/PowerShell-Docs/pull/11951/files


Not relevant to this PR, just thought I'd share